html/semantics/document-metadata/the-meta-element/pragma-directives/attr-meta-http-equiv-refresh/allow-scripts-flag-changing-1.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Meta refresh is blocked by the allow-scripts sandbox flag at its creation time, not when refresh comes due</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/semantics.html#attr-meta-http-equiv-refresh">
+
+<iframe id="source" src="support/refresh.sub.html?input=1%3B%20url%3Dfoo" sandbox="allow-same-origin"></iframe>
```

Comment:
That seems bad because then they navigate from about:blank to the new URL. Maybe instead onload="" content attributes is the way to go.